test/conftest.py

```
@@ -37,3 +43,114 @@
+                return False
+
+        # Poll the RTSP server until it's ready
+        max_attempts = 60
+        delay = 1
+        for attempt in range(max_attempts):
+            if is_rtsp_stream_ready(rtsp_address):
+                print("RTSP stream is ready.")
+                break
+            else:
+                print(
+                    f"Attempt {attempt + 1}/{max_attempts}: Stream not ready. Waiting {delay} seconds..."
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)